rs/test_utilities/execution_environment/src/lib.rs

```diff
@@ -1032,6 +1032,10 @@ impl ExecutionTest {
         self.canister_from_cycles_and_wat(self.initial_canister_cycles, wat)
     }
 
+    pub fn empty_canister(&mut self) -> CanisterId {
```

Comment:
Nit: there's already a function `pub fn create_canister(&mut self, cycles: Cycles) -> CanisterId` so maybe this function could be called `pub fn create_canister_with_default_cycles(&mut self) -> CanisterId`.